diff --git a/node_modules/react-native-root-toast/lib/ToastContainer.js b/node_modules/react-native-root-toast/lib/ToastContainer.js
index f5764bf..902cf31 100644
--- a/node_modules/react-native-root-toast/lib/ToastContainer.js
+++ b/node_modules/react-native-root-toast/lib/ToastContainer.js
@@ -36,13 +36,13 @@ const styles = StyleSheet.create({
         left: 0,
         right: 0,
         justifyContent: 'center',
-        alignItems: 'center'
+        alignItems: 'center',
     },
     containerStyle: {
         padding: 10,
         backgroundColor: '#000',
         opacity: 0.8,
-        borderRadius: 5
+        borderRadius: 5,
     },
     shadowStyle: {
         shadowColor: '#000',
@@ -279,6 +279,11 @@ class ToastContainer extends Component {
                     pointerEvents="none"
                     ref={ele => this._root = ele}
                 >
+                    <View style={{flexDirection: 'row', marginHorizontal: 16}}>
+                        <View style={{backgroundColor: 'white', alignItems: 'center', marginHorizontal: 12, width: 24, height: 24, borderRadius: 12}}>
+                            <Text style={{color: '#F04438', fontFamily: 'PoppinsSemiBold',
+        fontSize: 16, lineHeight: 28}}>!</Text>
+                        </View>
                     <Text style={[
                         styles.textStyle,
                         props.textStyle,
@@ -286,6 +291,8 @@ class ToastContainer extends Component {
                     ]}>
                         {this.props.children}
                     </Text>
+                    </View>
+                    
                 </Animated.View>
             </Touchable>
         </Wrapper> : null;
